### PR TITLE
Add custom annotations

### PR DESCRIPTION
**Pull request by kflorianski-ibm**
Also includes operator-sdk and controller-gen versions bump

### CODE REVIEW

File: controllers/resources/helper.go [hunk 1]
```diff
@@ -60,9 +62,7 @@ var (
 	IsServiceCAAPI             = true
 	IsAlertingEnabledByDefault = true
 	RHMPEnabled                = false
-	IsUIEnabled                = false
 	IsODLM                     = true
-	UIPlatformSecretName       = "platform-oidc-credentials"
```
Comment (ljeda) [inline controllers/resources/helper.go:65]: why do we remove IsUIEnabled and UIPlatformSecretName variables? as I've checked the repo, looks like we make references to them.

Comment (kflorianski-ibm) [reply controllers/resources/helper.go:65]: @ljeda Are you sure you checked on master branch?

Comment (kflorianski-ibm) [reply controllers/resources/helper.go:65]: See top right corner of each code block (`latest`):
image

File: bundle/metadata/annotations.yaml [hunk 2]
```diff
@@ -3,10 +3,10 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: ibm-licensing-operator-app
```
Comment (pgodowski) [inline bundle/metadata/annotations.yaml:6]: are we changing the name of the operator package?
if so, please DONT do this!

**Comment by kflorianski-ibm** [comment]
Split into 2 PR-s

**Pull request closed without merging**